### PR TITLE
add a third page+js

### PR DESCRIPTION
**Pull request by olya1998**
add third super puper page with JS implementation.
Plz, take a look :)

### CODE REVIEW

File: src/css/slick.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
+/* Slider */
+.slick-slider
+{
+    position: relative;
+
```
Comment (setysoft) [inline src/css/slick.css:5]: delete empty rows.plz

File: src/css/slick.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    -khtml-user-select: none;
+    -ms-touch-action: pan-y;
+        touch-action: pan-y;
+    -webkit-tap-highlight-color: transparent;
+}
+
+.slick-list
+{
+    position: relative;
+
```
Comment (setysoft) [inline src/css/slick.css:24]: and here

File: src/js/script.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+$(document).ready(function(){
+    $('.carousel__inner').slick({
```
Comment (setysoft) [inline src/js/script.js:2]: why __. you are using double underscore ?

**Review by setysoft — CHANGES_REQUESTED** [review]

**Review by setysoft — APPROVED** [review]

File: src/css/slick.css (hunk 1 above)
Comment (olya1998) [reply src/css/slick.css:5]: fixed

File: src/css/slick.css (hunk 2 above)
Comment (olya1998) [reply src/css/slick.css]: fixed

**Comment by olya1998** [comment]
fixed

**Pull request merged**